app/config/question-bank.js

```diff
@@ -1501,7 +1533,9 @@ const questionBank = {
               content: [{
                 para: `Robotic items must fit all 4 criteria to be eligible.\n\n
```

Comment:
These two need to be updated (grants-3058)

Reply:
Thanks Zac! Accidentally accepted the wrong change. Fixed now